docs/guides/api/index.rst

```diff
@@ -2,6 +2,4 @@
 API aspect
 ##########
 
-.. warning::
-
-   This page is a stub that we're working on.
+Please visit the Rubin Observatory Data Preview 0 (DP0) documentation for an `Introduction to the RSP API Aspect <https://dp0-2.lsst.io/data-access-analysis-tools/api-intro.html>`_.
```

Comment:
Sure, let's do this for now. Long term we need to move platform documentation out of the data release docs and into the platform/software docs.

Reply:
✔️